src/tests/color-spectrum.test.js

```diff
@@ -2,10 +2,23 @@ var should = require('should');
 var React = require('react/addons');
 var ColorSpectrum = require('../components/color-spectrum/color-spectrum.jsx');
 
+// https://github.com/facebook/react/issues/3721#issuecomment-106318499
+function shallowRender(makeComponent, context) {
```

Comment:
This is annoying. See https://github.com/facebook/react/issues/3721#issuecomment-106318499 for more details on why we need to do this.

I think this is an OK fix for right now, but ideally we might want to just migrate to a real phantom-based tester like teach.mozilla.org uses, rather than using [shallow rendering](http://facebook.github.io/react/docs/test-utils.html#shallow-rendering), which the docs claim is still experimental and lacks robust support for a number of things (like refs and contexts). I've just filed #311 for this.